Documentation/mkdocs.yml

```diff
@@ -28,6 +28,7 @@ nav:
     - The Control Input: 'the-control-input.md'
     - Three Dimensional Hexahedral Meshes: 'three-dimensional-hexahedral-meshes.md'
     - The Control File: 'the-control-file.md'
+    - The ISM Mesh file format: 'TheISMMeshFileFormats.md'
```

Comment:
```suggestion
    - The ISM Mesh File Format: 'TheISMMeshFileFormats.md'
```

To be consistent with title case used in the other headings.

Reply:
Fixed. Also the title has been changed to HOHQMesh instead of ISM, with appendix material moved to this doc.